mempool/src/tests/core_mempool_test.rs

```diff
@@ -141,7 +141,7 @@ fn test_update_transaction_in_mempool() {
     );
     let fixed_txns = add_txns_to_mempool(&mut mempool, vec![TestTransaction::new(0, 0, 5)]);
 
-    // check that first transactions pops up first
+    // Check that first transactions pops up first
```

Comment:
Missed some periods :)

Tbh, capitalization, and periods are a sort of personal, opinionated thing when it comes to comments.  I definitely think at least multi sentence comments need periods, and capitals (like English).  Though, it's much more consistent now!

Reply:
Agreed -- I was sort of just going through them mindlessly. But they should hopefully more consistent 😄